src/handlers/showcase.ts

```diff
@@ -12,8 +12,17 @@ export default async function handleShowcaseMessage(
     const showcaseChannel = (await client.channels.fetch(
       config.FORTIES_SHOWCASE
     )) as TextChannel;
+    let url;
 
-    const url = msg.attachments.first()?.url as string;
+    if (msg.attachments.size > 1) {
+      const acceptedFileFormats = new RegExp('.(jpe?g|png|gif|bmp)$', 'i');
```

Comment:
We should also support `webp`, `tif`, and `tiff` as accepted filetypes.

Reply:
I'll add those, wasn't sure if we wanted to support them or not so I left them out originally, but easy enough change.